builtin/logical/aws/secret_access_keys.go

```diff
@@ -286,6 +287,23 @@ func (b *backend) secretAccessKeysCreate(
 		}
 	}
 
+	if len(role.IAMTags) > 0 {
+		var tags []*iam.Tag
+		for key, value := range role.IAMTags {
+			k, v := key, value
```

Comment:
Might want to add a comment on why this is necessary, since it's a go-ism that isn't super comment, but is subtle and tricky to debug if you forget to do it.

Reply:
Commented